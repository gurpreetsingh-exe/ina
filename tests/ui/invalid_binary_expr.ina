// COMPILE_FAIL

fn main() {
    let a = 20;
    let b = 50.;
    let c = a + b;
}
