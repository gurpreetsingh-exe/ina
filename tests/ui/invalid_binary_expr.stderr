error: mismatch types
   ╭─[tests/ui/invalid_binary_expr.ina:6:13]
   │
 6 │     let c = a + b;
   │             ──┬──
   │               ╰─── cannot add `integer` and `float`
   ∙
───╯
